### DISCUSSION TITLE
I'm trying to recover my password, but I'm not receiving anything in my email.

### DISCUSSION THREAD

**Discussion by Kizat** [supabase/supabase#12824] [General]
I click on "forgot my password", I enter my mail, then supabase writes "The password reset email was successfully sent! Please check your mail", I check the mail, but there is no letter, and there is no letter in spam.

**Comment by kangmingtay** [comment]
@Kizat Can you please check if you had used github as your sign-in method? Password reset emails will only be sent to accounts created through email-password.
